torchci/components/GroupJobConclusion.tsx

```diff
@@ -94,47 +102,68 @@ function GroupTooltip({
   erroredJobs: JobData[];
   pendingJobs: JobData[];
 }) {
-  if (conclusion === "failure") {
+  if (conclusion === JobStatus.Failure) {
     return (
-      <div>
-        {`[${conclusion}] ${groupName}`}
-        <div>The following jobs errored out:</div>
-        {erroredJobs.map((erroredJob, ind) => (
-          <a
-            key={ind}
-            href={erroredJob.htmlUrl}
-            target="_blank"
-            rel="noreferrer"
-          >
-            {erroredJob.name}
-          </a>
-        ))}
-      </div>
+      <JobToolTip
+        conclusion={conclusion}
+        groupName={groupName}
+        jobs={erroredJobs}
+        message={"The following jobs errored out:"}
+      />
+    );
+  } else if (conclusion === JobStatus.Pending) {
+    return (
+      <JobToolTip
+        conclusion={conclusion}
+        groupName={groupName}
+        jobs={pendingJobs}
+        message={"The following jobs are still pending:"}
+      />
     );
-  } else if (conclusion === "pending") {
+  } else if (conclusion === JobStatus.Neutral) {
     return (
       <div>
         {`[${conclusion}] ${groupName}`}
-        <div>The following jobs are still pending:</div>
-        {pendingJobs.map((pendingJob, ind) => {
-          return (
-            <a
-              key={ind}
-              href={pendingJob.htmlUrl}
-              target="_blank"
-              rel="noreferrer"
-            >
-              {pendingJob.name}
-            </a>
-          );
-        })}
+        <div>All jobs were skipped</div>
       </div>
     );
   }
   return (
     <div>
       {`[${conclusion}] ${groupName}`}
-      <div>All relevant jobs passed</div>
+      <div>All jobs passed</div>
+    </div>
+  );
+}
+
+function JobToolTip({
```

Comment:
`JobTooltip` is already the name of a component I believe; maybe `GroupedJobTooltip` is better?